lucene/misc/native/src/main/posix/NativePosixUtil.cpp

```diff
@@ -102,6 +102,7 @@ JNIEXPORT jint JNICALL Java_org_apache_lucene_misc_store_NativePosixUtil_posix_1
 }
 #endif
 
+// TODO: To be removed together?
```

Comment:
This should really be a `// nocommit` right?  I.e. we want to solve this question before committing, and we don't plan to commit this `// TODO`.

Reply:
I'd like to get rid of it as soon as possible. This class is obsolete and completely unused, unless we also keep the original directory implementation.

Reply:
> This should really be a // nocommit right? I.e. we want to solve this question before committing, and we don't plan to commit this // TODO.

Ops sorry, didn't realize the nocommit usage there. Will use it for future patches.

> I'd like to get rid of it as soon as possible. This class is obsolete and completely unused, unless we also keep the original directory implementation.

I've removed them in latest commit.

Reply:
Awesome!

Yeah, `// nocommit` comment is really helpful in patches/PRs to note something that you know you need to fix before pushing.  Plus, our build tooling will catch us if we accidentally push code with `// nocommit` still.